runtime/src/local_runqueue_minheap.c

```diff
@@ -34,8 +36,13 @@ void
 local_runqueue_minheap_add(struct sandbox *sandbox)
 {
 	int return_code = priority_queue_enqueue_nolock(local_runqueue_minheap, sandbox);
-	/* TODO: propagate RC to caller. Issue #92 */
-	if (return_code == -ENOSPC) panic("Thread Runqueue is full!\n");
+	if (unlikely(return_code == -ENOSPC)) {
+		struct priority_queue *temp = priority_queue_grow_nolock(local_runqueue_minheap);
```

Comment:
It's because realloc returns `NULL` in case of error, but it doesn't free the original memory allocation, which would lead to a memory leak. I think it's a bit of a moot point since we likely would just want to panic the runtime if realloc fails, but still might be good for debugging in gdb.

Reply:
That's what I thought when seeing panic. But debugging point makes sense. Thank you!